pages/services/edge-lb/1.2/input-formats/index.md

```diff
@@ -9,22 +9,10 @@ enterprise: false
 ---
 
 
-Edge-LB accepts configuration files in either YAML or JSON format.
+Edge-LB accepts configuration files in either YAML or JSON format. Users are encouraged to use JSON and migrate their configuration from YAML format though, as YAML will be at some point deprecated.
```

Comment:
What do you think of:

`Edge-LB configuration files should preferably be written in JSON format. YAML is historically supported, but it will be deprecated in the future. We recommend users to migrate their YAML files to JSON`

Reply:
The reason is the user scheming for the content, read that YAML is supported first than JSON, and go write his/her conf in YAML.